### PR TITLE
Update deps with dashcore-lib 0.16.5

### PR DESCRIPTION
**Pull request by Cofresi**
Once this is merged we can prepare the release merge of v.5 to master

Right now waiting on latest dashcore-p2p to be ready.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -5,9 +5,9 @@
   "requires": true,
   "dependencies": {
     "@dashevo/dashcore-lib": {
-      "version": "0.16.4",
-      "resolved": "https://registry.npmjs.org/@dashevo/dashcore-lib/-/dashcore-lib-0.16.4.tgz",
-      "integrity": "sha512-uEkSBEI8LMoyPFzzjPghvBbkQpoVEL5yp6/r3ulAd0oTecdBH8gruwvsa9z4NlFBNO7/XcG5MVim3K8ZT7KmEA==",
+      "version": "0.16.3",
```
Comment (nmarley) [inline package-lock.json]: Should the version in this file be `0.16.3` when the version in `package.json` is `0.16.5`? I might be missing something, but seems like maybe a discrepancy.

Comment (Cofresi) [reply package-lock.json]: yep, totally right. Will update package-lock.json as soon as we have dashcore-p2p v.1.1.9 with the new dashcore-lib 0.16.5 on npm. Waiting for this to be merged https://github.com/dashevo/dashcore-p2p/pull/11

Comment (antouhou) [reply package-lock.json]: Published 1.1.9 on the npm

**Comment by antouhou** [comment]
I've also updated dependencies

**Review by antouhou — APPROVED** [review]
utACK

**Pull request merged**